src/coreclr/vm/fcall.h

```diff
@@ -1335,7 +1335,10 @@ typedef UINT32 FC_UINT8_RET;
 typedef INT32 FC_INT16_RET;
 typedef UINT32 FC_UINT16_RET;
 
+// Small primitive args are not widened.
```

Comment:
The comment around line 150 in this file still mentions CLR_BOOL.

Reply:
Mind if I update that in a separate change tomorrow?

Reply:
Fine with me

Reply:
Nevermind, updated it as well now due to those other places I missed the conversion.